key,default_value
additionalEventData_SignatureVersion,"SigV4"
additionalEventData_CipherSuite,"ECDHE-RSA-AES128-GCM-SHA256"
additionalEventData_AuthenticationMethod,"AuthHeader"
additionalEventData_x-amz-id-2,"rk2KewtoArmMz0aujqiUXyo2Am8Vj7b+1PpJNG8EvHOgrdlH0uFYJb9+qWfx47rLO99+WwHciG8="
userAgent,"[Boto3/1.14.51 Python/3.8.5 Linux/4.19.76-linuxkit Botocore/1.17.51]"
eventID,"5d5cf1b0-0d2e-479b-aaf4-863a64084c6f"
errorMessage,"The specified bucket does not exist"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDADO2GQD0K8TEF7KW1V"
userIdentity_arn,"arn:aws:iam::811596193553:user/Level6"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA3Z2XBVUDFQ9TU4MD"
userIdentity_userName,"Level6"
eventType,"AwsApiCall"
errorCode,"NoSuchBucket"
sourceIPAddress,"9.240.250.1"
eventName,"GetAccelerateConfiguration"
eventSource,"s3.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters_accelerate,""
requestParameters_bucketName,"dummy_data"
requestParameters_Host,"s3.amazonaws.com"
awsRegion,"us-east-1"
readOnly,"True"
requestID,"668429B588DBBB770"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2020-09-21T04:27:57Z"
